financialaid/constants.py

```diff
@@ -222,3 +222,19 @@
     "UG": 0,
     "ZW": 0
 }
+
+FINANCIAL_AID_APPROVAL_MESSAGE_BODY = """Your application for financial aid has been approved!
+Please log on to your account for more details.
+Love,
+MIT Micromasters"""
+FINANCIAL_AID_REJECTION_MESSAGE_BODY = """Your application for financial aid has been rejected.
+Please log on to your account for more details.
+Sympathy,
```

Comment:
Can we remove the `Sympathy` and `Love` here? I'm just looking to guard against the case where we forget to update this in a later issue